src/main/kotlin/com/bugsnag/android/gradle/internal/NdkToolchain.kt

```diff
@@ -129,17 +129,17 @@ abstract class NdkToolchain {
          * Minimum `bugsnag-android` version where the new symbol uploading is available, using `objcopy` to produce
          * the symbol files instead of `objdump`
          */
-        internal val MIN_BUGSNAG_ANDROID_VERSION = VersionNumber.version(5, 26)
+        internal val MIN_BUGSNAG_ANDROID_VERSION = Version(5, 26, 0)
```

Comment:
nit: mix of styles to create a `Version` compared with internal objects above (e.g. `VERSION_6_6`) - parse vs constructor.